javaslang/src/main/java/javaslang/Value.java

```diff
@@ -1225,11 +1234,8 @@ default CharSeq toCharSeq() {
 }
 
 interface ValueModule {
-
-    @SuppressWarnings("unchecked")
-    static <T extends Traversable<V>, V> T toTraversable(Value<V> value, T empty,
-            Function<V, T> ofElement,
-            Function<Iterable<V>, T> ofAll) {
+    static <T extends Traversable<V>, V>
+    T toTraversable(Value<V> value, T empty, Function<V, T> ofElement, Function<Iterable<V>, T> ofAll) {
```

Comment:
Mmhh, recently I applied our formatter to the whole code base.

Unifying is ok. For me it is important that the `static <generics> <return-type> <method-name>(` is kept in one line. The rest should be done by the formatter...